src/ride/track.c

```diff
@@ -4886,6 +4886,143 @@ void game_command_remove_track(int *eax, int *ebx, int *ecx, int *edx, int *esi,
  */
 void game_command_set_maze_track(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp)
 {
+	uint8 var_14 = 0x14;
+	uint8 var_10 = 0x10;
+	uint8 var_C = 0xC;
```

Comment:
It's best to understand what the register variables are first then it's easier to understand. Find where this game command is called to find out the variables. Ax is normally x, CX is normally y, BL is always flags, DX is normally z. Etc.